### PR TITLE
add beta notice to site and change site title

### LINKED ISSUE #31: Change the page title so it stops saying "Client"
lol

### PR DESCRIPTION
**Pull request by ackao**
image

the notice is ugly -- i can't figure out how to add line breaks. the following things did not work:
* `\n`
* `<p>line text</p>`
* putting double newlines in the text string
* `<br/>`

i'm out of ideas :(

fixes #31

### CODE REVIEW

File: client/src/app/notice/notice.component.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<nz-alert
+  nzType="error"
+  nzCloseable
+  nzMessage="This site is in BETA"
+  nzDescription="A DPS sim for SCH based on the original Google sheets sim is available: change stat values, party comp, and rotation info to see the effect on DPS.
+  Changes you make will NOT persist if you close the browser tab.
+  Future updates will allow gearsets to be saved/loaded, custom rotations, support for additional jobs, and more.
+  If you notice any issues or have feedback, please use the button in the lower left to visit us on GitHub, check to see if your issue is already tracked, and file one if it's not. Thanks!"
+></nz-alert>
```
Comment (Juanaxu) [inline client/src/app/notice/notice.component.html:9]: Looks like nzDescription either takes a string (that can't be formatted), or a TemplateRef (which is just a section of HTML).

Try putting this after the </nz-alert>:
<ng-template #alertDescription>
  
</ng-template>

And change the nzDescription to [nzDescription]="alertDescription"

Hopefully that works?

File: client/src/app/notice/notice.component.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+      nz-alert {
+        margin-bottom: 16px;
+      }
+
```
Comment (Juanaxu) [inline client/src/app/notice/notice.component.css:5]: nit: newline

File: client/src/app/notice/notice.component.html (hunk 1 above)
Comment (ackao) [reply client/src/app/notice/notice.component.html:6]: yep, it does! thanks!

**Comment by ackao** [comment]
image

**Review by Juanaxu — APPROVED** [review]

**Pull request merged**